{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1277, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1277, "_consider_importhook"]}
{"category": "PytestAssertRewriteWarning", "filename": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1277, "message": "Module already imported so cannot be rewritten: anyio", "$report_type": "WarningMessage", "when": "config", "location": ["/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/_pytest/config/__init__.py", 1277, "_consider_importhook"]}
{"pytest_version": "8.3.3", "$report_type": "SessionStart"}
{"nodeid": "tests57_76_5.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_76_5.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_76_5.py": 1, "Folder_76": 1, "": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00013867486268281937, "start": 1738461502.3971128, "stop": 1738461502.397252, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "734d47fd89294543ace4bcb7ea0e1e60", "node": "<WorkerController gw0>"}
{"nodeid": "tests57_76_5.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_76_5.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_76_5.py": 1, "Folder_76": 1, "": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 194, "message": "IndexError: index 1 is out of bounds for axis 0 with size 1"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[-255]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[-255]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_76/tests57_76_5.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    submatrix_sum = calculate_submatrix_sum(get_cumulative_sum(matrix), (i, j), (i + size, j + size))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_76/generated_answer.py", "lineno": 17, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return cumulative_matrix[bottom, right] - cumulative_matrix[top - 1, right] - cumulative_matrix[bottom, left - 1] + cumulative_matrix[top - 1, left - 1]"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_76/generated_answer.py", "lineno": 13, "message": "in calculate_submatrix_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 0 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[-255]])"], ["index", "(1, 1)"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 194, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[-255]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[-255]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_76/tests57_76_5.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    submatrix_sum = calculate_submatrix_sum(get_cumulative_sum(matrix), (i, j), (i + size, j + size))"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_76/generated_answer.py", "lineno": 17, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    return cumulative_matrix[bottom, right] - cumulative_matrix[top - 1, right] - cumulative_matrix[bottom, left - 1] + cumulative_matrix[top - 1, left - 1]"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/Phi_3_results_5/Folder_76/generated_answer.py", "lineno": 13, "message": "in calculate_submatrix_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 0 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[-255]])"], ["index", "(1, 1)"]]}, "reprlocals": null, "reprfileloc": {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 194, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/miniconda3/envs/benchmark/lib/python3.12/site-packages/numpy/matrixlib/defmatrix.py", "lineno": 194, "message": "IndexError: index 1 is out of bounds for axis 0 with size 1"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00023364927619695663, "start": 1738461502.397424, "stop": 1738461502.3976579, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "734d47fd89294543ace4bcb7ea0e1e60", "node": "<WorkerController gw0>"}
{"nodeid": "tests57_76_5.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_76_5.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_76_5.py": 1, "Folder_76": 1, "": 1}, "outcome": "passed", "longrepr": null, "when": "teardown", "user_properties": [], "sections": [], "duration": 0.00015219859778881073, "start": 1738461502.452697, "stop": 1738461502.4528499, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "734d47fd89294543ace4bcb7ea0e1e60", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
